### DISCUSSION TITLE
License Issue CompressRevision

### DISCUSSION THREAD

**Discussion by EdWeller** [ravendb/ravendb#18893] [General]
I went to update my community license for my development environment and it gives me an error about the Revision Compression.   It must have been some default that is no longer supported, because I did not set it and therefore do not know how to turn it off.   

Could someone please tell me how to turn this off so I may continue developing?  All my searches show a database record somewhere, but I do not know how to access it.   Thanks in advance.

**Comment by ppekrol** [comment]
From the code you can update this like follows:

```csharp
                var databaseRecord = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation("My_DB"));
                databaseRecord.DocumentsCompression.CompressRevisions = false;
                await store.Maintenance.Server.SendAsync(new UpdateDatabaseOperation(databaseRecord, databaseRecord.Etag));
```

From Studio, the database record for a particular database can be accessed via Settings -> Advanced -> Database Record

![image](https://github.com/user-attachments/assets/b87c7515-4214-4a85-a41b-2d8755c5e6eb)

Another option is to change the setting via dedicated view for compression. It can be found in Settings -> Document Compression -> Compress revisions for all collections

![image](https://github.com/user-attachments/assets/fb0be6e5-061e-449e-af7b-19308b05224f)

Please let us know if this helps

**Reply by EdWeller** [reply]
Thank You that served me well!
